conf/deploy.sh

```diff
@@ -14,6 +14,8 @@ npm install
 cd frontend
 ../node_modules/.bin/elm-package install -y
 cd ..
+
+npm run copyexamples
```

Comment:
In addition to this I think we need a way to upload the custom versions to server in Ansible. Otherwise custom stuff won't end up to server. We could do it like in conf/deploy.yml (upload static assets).